cmd/yorkie/project/update.go

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package project
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/yorkie-team/yorkie/admin"
+	"github.com/yorkie-team/yorkie/api/types"
+	"github.com/yorkie-team/yorkie/cmd/yorkie/config"
+)
+
+var (
+	flagAuthWebhookURL string
+	flagName           string
+	newName            string
+	newAuthWebhookURL  string
+)
+
+func newUpdateCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:     "update [name]",
+		Short:   "Update a project",
+		Example: "yorkie project update name [options]",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 1 {
+				return errors.New("name is required")
+			}
+
+			name := args[0]
+
+			token, err := config.LoadToken(config.AdminAddr)
+			if err != nil {
+				return err
+			}
+
+			cli, err := admin.Dial(config.AdminAddr, admin.WithToken(token))
+			if err != nil {
+				return err
+			}
+			defer func() {
+				_ = cli.Close()
+			}()
+
+			ctx := context.Background()
+			exists, err := cli.GetProject(ctx, name)
+			if err != nil {
+				return err
+			}
+			id := exists.ID.String()
+
+			if cmd.Flags().Lookup("name").Changed {
+				newName = flagName
+			} else {
+				newName = name
+			}
```

Comment:
Since `newName` is only used inside the RunE function, how about declaring it directly here?

And I think it's better to follow the style guide.
https://github.com/uber-go/guide/blob/master/style.md#unnecessary-else

```suggestion
			newName := name
			if cmd.Flags().Lookup("name").Changed {
				newName = flagName
			}
```

Reply:
Thank you for the good comment.
It's much cleaner.